upgrade-tests/src/test/java/io/zeebe/test/UpgradeTest.java

```diff
@@ -130,17 +135,23 @@
                 .deployWorkflow(
                     new Tuple<>(parentWorkflow(), PROCESS_ID),
                     new Tuple<>(childWorkflow(), CHILD_PROCESS_ID))
+                .createInstance()
                 .afterUpgrade(
-                    (state, wfKey, key) ->
-                        state
-                            .client()
-                            .newCreateInstanceCommand()
-                            .bpmnProcessId(PROCESS_ID)
-                            .latestVersion()
-                            .variables(Map.of("key", "123"))
-                            .send()
-                            .join()
-                            .getWorkflowInstanceKey())
+                    (state, wfKey, key) -> {
+                      final ActivatedJob job =
+                          state
+                              .client()
+                              .newActivateJobsCommand()
+                              .jobType(TASK)
+                              .maxJobsToActivate(1)
+                              .send()
+                              .join()
+                              .getJobs()
+                              .iterator()
+                              .next();
+
+                      state.client().newCompleteCommand(job.getKey()).send().join();
```

Comment:
We should verify that the child and parent instances are completed.

Reply:
The tests always check that the PROCESS_ID instance completes and that can't complete without the child process completing first right?